### ISSUE TITLE
508-defect-2 [HEADINGS, COGNITION]: Review page, headings should be logically ordered and consistently used

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22880]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect headings to be logically ordered and consistently used so I can confidently navigate a page.

## Details

The reviews page's headings needs review. Let's talk over a call to resolve this.

## Steps to Recreate

Review page

## Screenshots or Trace Logs

<img width="1297" alt="Screen Shot 2021-04-07 at 4 12 39 PM" src="https://user-images.githubusercontent.com/14154792/113928256-88d15c80-97bc-11eb-922f-5ec562479b45.png">

**Comment by steele-lm** [comment]
Possibly duplicates #64358

**Comment by zack** [comment]
This was completed as part of #64130

PR: https://github.com/department-of-veterans-affairs/vets-website/pull/25896
